docs/2.guide/2.directory-structure/2.env.md

```diff
@@ -0,0 +1,24 @@
+---
+navigation.icon: IconFile
+title: ".env"
+description: "A .env file specifies your build/dev-time environment variables."
+head.title: ".env"
+---
+
+# .env File
+
+Nuxt CLI has built-in [dotenv](https://github.com/motdotla/dotenv) support in development mode and when running `nuxi build` and `nuxi generate`.
+
+In addition to any process environment variables, if you have a `.env` file in your project root directory, it will be automatically loaded **at build, dev, and generate time**, and any environment variables set there will be accessible within your `nuxt.config` file and modules.
+
+::alert
+If you want to use a different file - for example, to use `.env.local` or `.env.production` - you can do so by passing the `--dotenv` flag when using `nuxi`. Just as above, this will only apply in development mode and when running `nuxi build` and `nuxi generate`.
+::
+
+When updating `.env` in development mode, the Nuxt instance is automatically restarted to apply new values to the `process.env`.
```

Comment:
Small note, "unsetting" a variable would not automatically take effect. This might be fixed somehow latet is a current behavior. (same goes with removing entier file)